www/docs/usage/faker.md

```diff
@@ -2,5 +2,67 @@
 title: Faker Integration
 description: |
   How to use the faker.js, the most popular data generation framework for
-  generating networks of data
+  generating networks of data.
 ---
+
+**[Faker.js][faker] has most of the [custom generations][custom-generators]
+you'll ever need**
+
+You will rarely write your own custom generators for two reasons. The first is
+that it is tedious, and the second is that most of them have already been
+written, and furthermore, are being actively maintained by open source
+communities.
+
+[Faker][faker] is a very complete, very mature, and very well maintained
+generator of mock data which can be easily slotted into the role of generating
+the field data for your graph.
+
+There is not currently an off the shelf library to do this for you, but this
+recipe is easy to follow and will have you generating "massive amounts of fake
+data" in no time.
+
+The strategy is three fold. First we will collect all of the
+generation methods in the entire Faker library as and put them under
+the `@faker/` namespace, and second, we'll make a very common list of
+patterns for accessing those methods. Here's our faker generator:
+
+``` javascript
+import { faker as globalFaker, Faker } from "@faker-js/faker";
+import { createDispatch } from "@frontside/graphgen";
+
+const methods = Object.entries(globalFaker).reduce((methods, [name, mod]) => {
+  if (mod) {
```

Comment:
Does 'faker' really contain 'falsy' values?

Reply:
It's to filter out faker "modules" that don't have any generator content, so mostly there is `faker.internet`, and `faker.address` and `faker.datatype` etc... but there are also properties just "hanging" on `faker` that don't represent a collection of generators like `faker.fakerStatic`

Reply:
Although it does seem weird.....